### PR TITLE
More heading fixes to budget and business section

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
## Changes
- Missed a couple pages with the old html5 headings, this fixes those pages

## Notes
- I'd like to get this in after #992 #995, since I'm sure it'll change some of the selectors there, so no rush to review.

### CODE REVIEW

File: src/budget/index.html [hunk 1]
```diff
@@ -121,11 +121,11 @@ <h2 class="h3">Funding requests</h2>
                     block__border-right">
             <div class="content-l content-l__main">
                 <section class="content-l_col content-l_col-1-2">
-                    <h1 class="header-slug">
+                    <h2 class="header-slug">
```
Comment (anselmbradford) [inline src/budget/index.html]: This looks like it would be a style change potentially. Is `/budget/` the same on flapjack.demo vs local?

Comment (KimberlyMunoz) [inline src/budget/index.html]: Yes. `header-slug` controls the styles and size. The heading is just for semantics.

Comment (anselmbradford) [inline src/budget/index.html]: Cool, thanks for verifying!

**Comment by anselmbradford** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**